tests/helper/util.ts

```diff
@@ -101,7 +101,7 @@ export function getTrailingSpaces(string: string): string {
 }
 
 export function formatHtml(html: string): string {
-	return Prettier.format(html, {
+	return synchronizedPrettier.format(html, {
 		printWidth: 100,
 		tabWidth: 4,
 		useTabs: true,
```

Comment:
Non-blocking, but it would be nice if we could import our Prettier config here (and remove the plugins via JS since they don't work with sync prettier), instead of duplicating it.